sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
 # Release History
 
-## 7.0.0-beta.7 (Unreleased)
+## 7.0.0-beta.7 (2020-11-06)
+### New Features
+- Added automatic message and session lock renewal feature on the receiver clients. By default, this will be done 
+  for 5 minutes.
+- Added auto complete feature to the async receiver clients. Once the client completes executing the user provided 
+  callback for a message, the message will be completed. If the user provided callback throws an error, the message 
+  will be abandoned.
+- An intermediate SessionReceiverClient is introduced to act as the factory which can then be used to accept sessions 
+  from the service. Accepting a session would give you the familiar receiver client tied to a single session.
+- Added ServiceBusProcessorClient which takes your callbacks to process messages and errors in an infinite loop. This 
+  also supports working with sessions where you can provide the maximum number of sessions to work with concurrently. 
+  When the client no longer receives any messages from one session, it rolls over to the next available session.
 
+### Breaking Changes
```

Comment:
I remember two break changes related to session. Not sure about others.
We removed all methods that have sessionId as an argument on ServiceBusReceiverClient and the async client.
The client builder after receiver() removed sessionId and maxConcurrentSessions